hevm/index.html

```diff
@@ -26,34 +21,61 @@ <h1>
 
 <pre class="small">
 COMMANDS
-
-<strong>$ hvem dapp-test</strong>            run dappsys unit tests
-<strong>$ hvem interactive</strong>          enter the interactive debugger
-
-OPTIONS
-
---json-file      run against a specific output file
---rpc            fetch external contracts via rpc from the latest block
-</pre>
+<code>
+hevm -- Ethereum evaluator
+
+Usage: hevm (exec | dapp-test | interactive | bc-test | compliance | flatten |
+            emacs | version | rlp | abiencode | merkle-test | strip-metadata)
+
+Available options:
+  -h,--help                Show this help text
+
+Available commands:
+  exec
+  dapp-test
+  interactive
+  bc-test
+  compliance
+  flatten
+  emacs
+  version
+  rlp
+  abiencode
+  merkle-test
+  strip-metadata
```

Comment:
I don't know if this is useful to have on the website, but I'm fine if you want to add it.